single_chart_migration/mongo.yaml

```diff
@@ -179,6 +179,8 @@ spec:
               value: "no"
             - name: MONGODB_ENABLE_DIRECTORY_PER_DB
               value: "no"
+            - name: MONGODB_EXTRA_FLAGS
```

Comment:
This change will also affect the final upgrade right.
You want it just for trial correct?

Reply:
In the trial run, this mongo.yaml gets used and once the migration is complete, the mongo resources including PVCs get deleted. 

For actual migration (which uses the helm package), the mongo cache size is already available as a helm param and the it can be configured. The default value for that is 4GB.